apps/mull-api/src/app/media/media.resolver.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { Media } from '../entities';
+import { mockFile, mockInvalidFile } from './media.mockdata';
+import { MediaResolver } from './media.resolver';
+import { MediaService } from './media.service';
+import { FileUpload } from 'graphql-upload';
+import { createWriteStream, renameSync } from 'fs';
+import { join } from 'path';
+import fs = require('fs');
+
+const mockMediaService = () => ({
+  create: jest.fn((mockMimeType: string) => {
+    const mockFileType = mockMimeType.split('/')[1];
+    const mockMedia = new Media(mockFileType);
+    return mockMedia;
+  }),
+  saveFile: jest.fn((mockFile: FileUpload) => {
+    return new Promise((resolve, reject) => {
+      mockFile.createReadStream().pipe(
+        createWriteStream(join(process.cwd(), `apps/mull-api/uploads/${mockFile.filename}`))
+          .on('finish', () => resolve(true))
+          .on('error', () => {
+            reject(false);
+          })
+      );
+    });
+  }),
+  updateFilename: jest.fn(
+    (mockPrevFilename: string, mockNextFilename: number, mockFileType: string) => {
+      try {
+        renameSync(
+          join(process.cwd(), `apps/mull-api/uploads/${mockPrevFilename}`),
+          join(process.cwd(), `apps/mull-api/uploads/${mockNextFilename}.${mockFileType}`)
+        );
+      } catch (err) {
+        console.log(err);
+        throw err;
+      }
```

Comment:
You have a try-catch where after catching an error, you throw it right back. You don't even need the try-catch in that case.
```suggestion
        renameSync(
          join(process.cwd(), `apps/mull-api/uploads/${mockPrevFilename}`),
          join(process.cwd(), `apps/mull-api/uploads/${mockNextFilename}.${mockFileType}`)
```